lib/rspec/core/formatters/exception_presenter.rb

```diff
@@ -244,7 +244,7 @@ def formatted_message_and_backtrace(colorizer)
         end
 
         def exception_backtrace
-          exception.backtrace || []
+          RSpec.configuration.backtrace_formatter.format_backtrace(exception.backtrace) || []
```

Comment:
Real example. I have helper method in support which is waiting for passed block to return true. Where is nothing useful in the code itself when I see a failure, but in the test I have useful information about it, like `waiter { condition_to_make_me_happy }`. With current functionality, the only way to find out this condition is to open the file on the line from backtrace. With proposed way this line will be in snippet.
But